docs/source/basic_tutorials/consuming_tgi.md

```diff
@@ -1,81 +1,117 @@
 # Consuming Text Generation Inference
 
-There are many ways you can consume Text Generation Inference server in your applications. After launching, you can use the `/generate` route and make a `POST` request to get results from the server. You can also use the `/generate_stream` route if you want TGI to return a stream of tokens. You can make the requests using the tool of your preference, such as curl, Python or TypeScrpt. For a final end-to-end experience, we also open-sourced ChatUI, a chat interface for open-source models.
+There are many ways to consume Text Generation Inference (TGI) server in your applications. After launching the server, you can use the [Messages API](https://huggingface.co/docs/text-generation-inference/en/messages_api) `/v1/chat/completions` route and make a `POST` request to get results from the server. You can also pass `"stream": true` to the call if you want TGI to return a stream of tokens. While `/generate` and `/generate_stream` are still available, the Messages API is recommended as it automatically applies the chat template.
```

Comment:
This paragraph introduces multiple concepts in parallel and I'm not sure it's the best for the introduction.

As a beginner
>  While `/generate` and `/generate_stream` are still available, the Messages API is recommended as it automatically applies the chat template.

Does not tell me much given I don't even know what this routes are. I would potentially remove

Reply:
Fixed in the latest commit.